views/fe-utils/fetchUtil.js

```diff
@@ -1,8 +1,14 @@
-import FileSaver from 'file-saver'
 import basePathConfig from '../../server/configs/basePathConfig'
```

Comment:
I was about to comment on this being a weird file to hold model logic, but I see you're reducing it dramatically here. Nice work!